### ISSUE TITLE
review Increased phosphoribosylpyrophosphate synthetase

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#4302]
https://hpo.jax.org/app/browse/term/HP:0003240

I thnk we we "activity". THere are no current annotations. Bu there should be increased/decreased, e.g., Charcot-Marie-Tooth disease-5

**Comment by pnrobinson** [comment]
see
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC427101/


COmment: intracellularavailability of the sugar phosphate, 5-phosphoribosyl 1-pyrophosphate (PP-ribose-P), is an important factor in regulation of the rate of purine synthesis de novo. PP-ribose- P synthetase (EC2.7.6.1) is an enzyme that catalyzes the synthesis of PP-ribose-P from ATP and ribose-5-phosphate. An increased activity of PP-ribose- P synthetase can be associated with increased PP-ribose-P concentrations and increased purine biosynthetic rates. Increased purine production in turn can be associated with gout. PMID:4200723

**Comment by pnrobinson** [comment]
On the other hand
PRS enzyme activity in three individuals with CMTX5 was decreased compared to controls [Kim et al 2007].

Note: Because it is difficult to assay PRS1 enzyme activity separately from that of the other two isoforms (PRS2 and PRS3), decrease in PRS enzyme activity is assumed to reflect decreased activity of PRS1, not PRS2 or PRS3. https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1950833/

see https://github.com/monarch-initiative/hpo-annotation-data/issues/315

**Comment by pnrobinson** [comment]
- [ ] Add term Decreased phosphoribosyl pyrophosphate synthetase activity
Definition: Abnormally reduced activity of the enzyme phosphoribosyl pyrophosphatesynthetase, which catalyzes the synthesis of PP-ribose-P from ATP and ribose-5-phosphate.  PMID:17701900 PMID:20301731
Comment: It is difficult to assay PRS1 enzyme activity separately from that of the other two isoforms (PRS2 and PRS3), decrease in PRS enzyme activity is assumed to reflect decreased activity of PRS1, not PRS2 or PRS3

**Comment by pnrobinson** [comment]
- [ ] Add term Abnormal phosphoribosyl pyrophosphate synthetase activity
def: Any deviation from the normal activity of the enzyme phosphoribosyl pyrophosphatesynthetase, which catalyzes the synthesis of PP-ribose-P from ATP and ribose-5-phosphate.
add the comment from the above
PMID:4200723
- [ ] Also modify the term label as suggested above.
